react/Text/Text.js

```diff
@@ -13,6 +13,8 @@ import stylesRegular from '../Regular/Regular.less';
 import withTextProps, { SizePropTypes } from '../private/withTextProps';
 
 const Text = ({
+  component,
```

Comment:
Can we add `component` to the proptypes definition below (along with the default)

Reply:
Added component proptype definition. Thanks